source/_patterns/01-atoms/list-group/demo/list-groups.twig

```diff
@@ -0,0 +1,209 @@
+{# This function loops through the scssColors global variable to make an array of all the available colors. #}
+{% set colors = [] %}
+{% for name, rgb in scssColors %}
+  {% set item_classes = {
+    item_color: name
+  } %}
+  {% set colors = colors|merge([name]) %}
+{% endfor %}
+
+
+<div class="container">
+  {% embed '@atoms/grid/_grid-1-up.twig' %}
+    {% block column_1 %}
+      <div class="mb-3">
+        <h1 class="text-center">List Group</h1>
+        {% include '@molecules/card/_card-helper.twig' with {
+          card_text: list_group_helper,
+        } %}
+      </div>
+
+      {# Basic examples #}
+      <div class="mb-3">
+        {% include '@molecules/card/_card-helper.twig' with {
```

Comment:
Want to remove `card-helper` styling wrapper styles right before each thing we demo. Too much dark filling up the page. We can leave the first, overall description (ie line 16).